tests/DeepCopyTest/Reflection/ReflectionHelperTest.php

```diff
@@ -29,6 +29,40 @@ public function testMaintainPropertiesKey()
 
         $this->assertSame($expectedProps, array_keys($actualProps));
     }
+
+    /**
+     * @dataProvider propertyDataProvider
+     *
+     * @param string $name
+     */
+    public function testGetProperty($name)
+    {
+        $object = new ReflectionHelperTestChild();
+        $property = ReflectionHelper::getProperty($object, $name);
+        $this->assertInstanceOf('\ReflectionProperty', $property);
+        $this->assertSame($name, $property->getName());
+    }
+
+    /**
+     * @return array
+     */
+    public function propertyDataProvider()
+    {
+        return [
+            'public property' => ['a4'],
+            'private property' => ['a9'],
+            'private property of ancestor' => ['a3']
```

Comment:
I agree with you, that I'm not supposed to access to the private properties of the parent entity. But  this possibility for copying private properties of ancestor objects was added [here](https://github.com/myclabs/DeepCopy/pull/23). Currently the `ReflectionHelper:getProperties()` return all the properties from parent classes  (include private from parent classes), but at the same time the Filters & Matchers is not support it, so in this PR I fixed BC break that was introduced after merge https://github.com/myclabs/DeepCopy/pull/23, keeping backward compatibility.